pyne/cpp_source_sampling.pxd

```diff
@@ -54,4 +56,4 @@ cdef extern from "source_sampling.h" namespace "pyne":
 
 
 
-{'cpppxd_footer': '', 'pyx_header': '', 'pxd_header': '', 'pxd_footer': '', 'cpppxd_header': '', 'pyx_footer': ''}
\ No newline at end of file
+{'cpppxd_footer': '', 'pyx_header': '', 'pxd_header': '', 'pxd_footer': '', 'cpppxd_header': '', 'pyx_footer': ''}
```

Comment:
Please remove this line

Reply:
This line is removed in #997 .

Reply:
hmm then you shouldn't modify it here to prevent conflicts.